cli_tools/gce_vm_image_import/cli/cli.go

```diff
@@ -34,17 +32,16 @@ import (
 // Main starts an image import.
 func Main(args []string, toolLogger logging.ToolLogger, workflowDir string) error {
 	logging.RedirectGlobalLogsToUser(toolLogger)
-
 	ctx := context.Background()
 
-	// 1. Parse the args without validating or populating. Splitting parsing and
-	// validation allows us to log the intermediate, non-validated values, if
-	// there's an error setting up dependencies.
-	importArgs, err := importer.NewImportArguments(args, files.MakeAbsolute(workflowDir))
+	// 1. Parse authentication-related arguments, which are required for parsing and
+	// validating the remaining arguments.
```

Comment:
What does it mean "authentication-related arguments" here? I don't see much authentication related in parseArgsFromUser()

Reply:
I can see how that's ambiguous. I updated with more context, explaining why parsing occurs in two phases (first to get auth overrides, second to fully verify and populate)